client/my-sites/post-type-list/post-actions-ellipsis-menu/view.jsx

```diff
@@ -42,6 +42,7 @@ class PostActionsEllipsisMenuView extends Component {
 
 		this.props.setPreviewUrl( previewUrl );
 		this.props.setLayoutFocus( 'preview' );
+		this.props.onClick();
```

Comment:
>Should I specify default props in `PostActionsEllipsisMenuView` then?

I don't know that we need all the props to have defaults provided; only those which could potentially cause errors if not passed as supported. Would be fine to leave as-is though.